client/trino-client/src/main/java/io/trino/client/OkHttpUtil.java

```diff
@@ -324,6 +332,25 @@ private static KeyStore loadTrustStore(File trustStorePath, Optional<String> tru
         return trustStore;
     }
 
+    private static KeyStore loadSystemKeyStore(Optional<String> keyStoreType)
+            throws IOException, GeneralSecurityException
+    {
+        String osName = Optional.ofNullable(StandardSystemProperty.OS_NAME.value()).orElse("");
+        Optional<String> systemKeyStoreType = keyStoreType;
+        if (!systemKeyStoreType.isPresent()) {
+            if (osName.contains("Windows")) {
+                systemKeyStoreType = Optional.of("Windows-MY");
```

Comment:
Consider adding a comment why `Windows-MY` is used here, where `Windows-ROOT` is used for the trust store. I tried looking it up myself, and found this comment: https://bugs.openjdk.org/browse/JDK-6782021?focusedId=14488495&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14488495
So maybe also consider `Windows-MY-CURRENTUSER`, which is more explicit.

Also, how about adding string constants for `Windows`, `Mac`, and those types of trust/key stores? They're used in multiple places now.

Reply:
I've moved to keystore constants for both OS' to the top of the class and added some code comments. I haven't linked to the actual docs since I didn't want to link to a specific Java version that'll become outdated over time.